### DISCUSSION TITLE
Feature request: Disable chat scrollbar option

### DISCUSSION THREAD

**Discussion by DataCrusade** [runelite/runelite#17381] [Ideas]
![image](https://github.com/runelite/runelite/assets/36621129/6a660988-9339-431e-9243-ba56e913448d)

It's inconvenient to misclick on the chat scrollbar and to have your click blocked by it and (less importantly) your chat scrolled up. A toggleable option to disable it would be reasonable and convenient to solve this issue. You would still be able to scroll your chat via your mouse's scroll wheel with the option enabled

**Comment by dabolink** [comment]
![image](https://github.com/runelite/runelite/assets/5316046/00c24bc9-7bf3-4324-ac76-061ff7d6be1e)
there is an option in game to move it to the left side. i found this pretty useful for removing misclicks

**Comment by DataCrusade** [comment]
> ![image](https://private-user-images.githubusercontent.com/5316046/296840517-00c24bc9-7bf3-4324-ac76-061ff7d6be1e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU0NTMwMTQsIm5iZiI6MTcwNTQ1MjcxNCwicGF0aCI6Ii81MzE2MDQ2LzI5Njg0MDUxNy0wMGMyNGJjOS03YmYzLTQzMjQtYWM3Ni0wNjFmZjdkNmJlMWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDExNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMTdUMDA1MTU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2U0NTNmMjdhMGM5YmNiZTJhYjhiZjc3M2NjNmJhZDdhYzI1OGI2NzIyMGNjMDlmZmQzODNjNjJkMWYzYzk1MyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.R3rcDo1Akn_SbFTNIGg9-mBb2ky5xiY3U8W_hNCXlbg) there is an option in game to move it to the left side. i found this pretty useful for removing misclicks

 I was not aware of this, thank you for the info. This suggestion probably isn't necessarily but might still be useful to some people
